src/IO/H5/Python/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+set(LIBRARY "PyIOH5File")
+
+spectre_python_add_module(
+  IOH5File
+  SOURCES Bindings.cpp Dat.cpp File.cpp
```

Comment:
Could you please list of source files on separate lines? Having them on separate lines reduces the likelihood of a conflict.

```
  SOURCES
  Bindings.cpp
  Dat.cpp
  File.cpp
```